[BUILD] Windows - Visual Studio 2019 : fatal error LNK1248

### Environment
**Operating System:** (e.g. Windows 2019, macOS-10.15, Linux CentOS 7.4)

Windows 10

**Version / Commit SHA:** (e.g VDB 7.2, SHA: 1a2b3c4d5e...)

OpenVDB 11.0.0

**CMake Version:** (e.g. 3.18)

3.25

**Compiler:** (e.g. gcc 10, msvc 2019)

VS 2019

### Describe the problem

Linker error

```
openvdb\openvdb\libopenvdb.lib : fatal error LNK1248: image size (1017E3216) exceeds maximum allowable size (FFFFFFFF)
[155/156] Building CXX object openvdb_cmd\vdb_print\CMakeFiles\vdb_print.dir\main.cc.obj
ninja: build stopped: subcommand failed.
```

vdb_print is not critical to me at this point so I can remove it from the build but reporting this anyway.

--- a/openvdb_ax/openvdb_ax/CMakeLists.txt
+++ b/openvdb_ax/openvdb_ax/CMakeLists.txt
@@ -306,6 +306,10 @@ if(OPENVDB_AX_STATIC)
     PROPERTIES OUTPUT_NAME openvdb_ax
   )
 
+  if(WIN32)
+    set_target_properties(openvdb_ax_static PROPERTIES PREFIX "lib")
+  endif()
+
   target_link_libraries(openvdb_ax_static PUBLIC ${OPENVDB_AX_CORE_DEPENDENT_LIBS})
 
   # @todo fix opaque pointer requirements

--- a/pendingchanges/ax_windows_cmake_fix.txt
+++ b/pendingchanges/ax_windows_cmake_fix.txt
@@ -1,0 +1,3 @@
+Build:
+    - Fixed an issue with OpenVDB AX's CMake on Windows where the static and shared library targets would have the same name
+    [Reported by Nicholas Yue]

